orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -531,7 +555,13 @@ void addObjectMethodsForTraining(py::module& m) {
                                                bw_fetches_names, bw_outputs_device_info);
       }))
       .def("run_forward", [](TrainingAgent* agent, const std::vector<OrtValue>& feeds, std::vector<OrtValue>& fetches, PartialGraphExecutionState* state) -> void {
```

Comment:
>def("run_forward", [](TrainingAgent* agent, const std::vector<OrtValue>& feeds, std::vector<OrtValue>& fetches, PartialGraphExecutionState* state) -> void { [](http://example.com/codeflow?start=7&length=156)

Can please get rid of this API and directly use the one below by passing empty cache from Python?

Reply:
see https://github.com/microsoft/onnxruntime/pull/8273#issuecomment-900515827